.travis.yml

```diff
@@ -2,22 +2,24 @@
 language: c++
 sudo: required
 
-os:
-  - linux
-  - osx
```

Comment:
is there a way to keep osx running as it has been and have the new docker based way for linux?